counter/src/main/java/org/infinispan/counter/impl/manager/EmbeddedCounterManager.java

```diff
@@ -44,8 +46,8 @@ public class EmbeddedCounterManager implements CounterManager {
    public static final String OBJECT_NAME = "CounterManager";
    private static final Log log = LogFactory.getLog(EmbeddedCounterManager.class, Log.class);
 
-   private final Map<String, Object> counters;
-   private volatile boolean started;
+   private final Map<String, CompletableFuture<InternalCounterAdmin>> counters;
```

Comment:
We need to do a null check now in the `getConfigurationAsync` method as the config may not be present.

Reply:
I'm sorry, I don't understand where I need a null check :(